pkg/types/conversion/installconfig.go

```diff
@@ -78,8 +78,8 @@ func convertNetworking(config *types.InstallConfig) {
 
 	// Recognize the default network plugin name regardless of capitalization, for
 	// backward compatibility
-	if strings.ToLower(netconf.NetworkType) == strings.ToLower(string(operv1.NetworkTypeOpenShiftSDN)) {
-		netconf.NetworkType = string(operv1.NetworkTypeOpenShiftSDN)
+	if strings.ToLower(netconf.NetworkType) == strings.ToLower(string(operv1.NetworkTypeOVNKubernetes)) {
+		netconf.NetworkType = string(operv1.NetworkTypeOVNKubernetes)
```

Comment:
This should not be changed. Well, the comment should be changed to not say "default", but the purpose of this code is not "whatever the default networkType value is, we should recognize it case-insensitively", it's "the correct value of this field used to be `OpenshiftSDN` but then we fixed it to be `OpenShiftSDN`, but we need to continue matching both values for backward-compatibility with (very) old install-configs.

Reply:
fixed